service/ui.go

```diff
@@ -86,6 +88,17 @@ func NewUIServer(settings *evergreen.Settings, queue amboy.Queue, home string, f
 			settings.Jira.GetHostURL(),
 			settings.Jira.Username,
 			settings.Jira.Password),
+		umconf: gimlet.UserMiddlewareConfiguration{
+			HeaderKeyName:  evergreen.APIKeyHeader,
+			HeaderUserName: evergreen.APIUserHeader,
+			CookieName:     evergreen.AuthTokenCookie,
+			CookieTTL:      365 * 24 * time.Hour,
```

Comment:
What happens after a year?

Reply:
It expires and you have to reauth. 

this preserves the current behavior: https://github.com/evergreen-ci/evergreen/blob/master/auth/auth.go#L50